app/services/move_events/params_validator.rb

```diff
@@ -6,9 +6,10 @@ class ParamsValidator
 
     ACCEPTABLE_PLURAL_VERBS = %w[cancels completes approves rejects].freeze
 
-    attr_reader :timestamp, :type, :date, :cancellation_reason, :rejection_reason
+    attr_reader :timestamp, :type, :date, :cancellation_reason, :rejection_reason, :from_location, :to_location
 
-    validates :type, presence: true, inclusion: { in: %w[cancel complete lockouts redirects approve reject accepts] }
+    # TODO: if/when pluralising types, existing move event records will need migrating
```

Comment:
sorry, is not clear to me what `pluralising types` mean, could you expand a bit on this? when do we need to do it?

Reply:
sorry that wasn't clear - I'll add a more detailed note to explain the situation